seedlink_plotter/seedlink_plotter.py

```diff
@@ -474,9 +497,61 @@ def get_events(self):
         with self.lock:
             start = min([tr.stats.starttime for tr in self.stream])
             end = max([tr.stats.endtime for tr in self.stream])
-        neries_emsc = Client("EMSC")
-        events = neries_emsc.get_events(starttime=start, endtime=end,
-                                          minmagnitude=self.args.events)
+        
+        events = Catalog()
+        
+         # get the coordinats of the station
+        splitted_ids = [x.split(".") for x in self.args.ids]  # use seedlink id to get network, station, location and channel
+        for client in ["iris-federator","eida-routing"]:
+            try:
+                Cli = RoutingClient(client)
```

Comment:
I don't like `Cli` as var name here. First, capitalized is for classes not variables. Second, `Cli` sounds like "command line interface". Please use `client` as var name instead.